Invert Fly.io's logo on all news posts + Added news page styling #354

Dhuds1 opened the pull request:
## Non merge related
Hello, I am really new to this community and I'm very much in love with y'alls! I'm currently in a college and our most recent assignment was to contribute to an open source community. I've been looking for a way to contribute to this community for a while, and I noticed a few things that I thought I could help with!

## Merge Related
### Fly.io logo to inverted version
issue #354 
I added the fly.io's inverted logo to the `images/sponsors folder`, and named it `fly-inverted.svg`. I went through each news post and updated the `fly.svg` to `fly-invert.svg`

### News.html styling
Afterwards, I thought that the news.html page was a little hard on the eyes, and that there was a lack of typographic hierarchy with the text which makes it hard to figure out which text is most to least important. I changed the `publish date, and author` text to`var(--color-code-grey)` and text size to `var(--font-size-small)`. Afterwards, I added a hover effect on the `{{ post.title }}` to help clarify what is a link. I then changed the `li { display }` to better space its content. I then added a `li::before` to create a bar of `var(--color-faff-pink)` to the left size of the articles. This helps "block" the articles, and add a little visual interest.  Oh, I forgot, I also added the same lucy-hero effect to the Lucy in the footer, for some ✨ Spice ✨

### News.html alterations
Inside the each post `<li>` I added custom data fields, `post-tags` and `post-author` with the idea that maybe later on you'd want to implement some sort of user-filtering. I also added an `<a href="{{ post.author-link }}">` tag to `{{ post.author }}` because I figured maybe someone would want to check out more about the author?

Sorry for the wall of text, this is my first time creating a PR and I wanted to be concise.

## TL:DR;
- made `fly.io` inverted on news articles
- added typographical hierarchy to posts in `news.html`
- added visual interest to posts in `news.html` 
- added `hover` on `{{ post-title }}`
- added `hover rotation and svg change` on `lucy` footer version
- added author link to `{{ post.author }}`
- added `post-tags ` and `post-author` data-attributes `into posts

lpil commented on news.html:
```diff
@@ -5,11 +5,12 @@
   {% for post in site.posts %}
-  <li>
-    <a href="{{ post.url }}"><h2>{{ post.title }}</h2></a>
+  <li post-tags="{% for tag in post.tags %}{{ tag }} {% endfor %}" post-author="{{post.author}}">
```
Let's remove these tags are we don't have a use for them yet

lpil commented on news.html:
```diff
@@ -5,11 +5,12 @@
     <p>{{ post.subtitle }}</p>
     {% endif %}
-    <span>Published {{ post.date | date: "%d %b, %Y" }} by {{ post.author }}</span>
+    <span>Published {{ post.date | date: "%d %b, %Y" }} by <a class="links author" target="_blank" href="{{post.author-link}}">{{ post.author }}</a></span>
```
Nice! Do you think it's worth not rendering the link if the author has no author-link?

lpil reviewed (commented):
Hello! Thank you for this, you've made some really nice improvements here. Normally I'd say it's good to have one PR per thing being changed to avoid one problem blocking the others, but I think we're all good here :)

I've left some very small notes inline, once resolved we can merge this!

Could you also do me a favour and reduce the amount of padding on the left of `.news-posts`? When checking out your nice addition to that page I noticed that the default ul padding makes it rather positioned, especially on mobile.

Thank you!

Dhuds1 commented on _layouts/layout.html:
```diff
@@ -1,4 +1,4 @@
-<!doctype html>
+<!DOCTYPE html>
```
oops my bad, I think it was my formatter. I'll revert it!

Dhuds1 commented:
I removed the padding in `li`, and moved the `li::before` `-20px` to the `left`. I also removed the `data-attributes` in the `li` and added a conditional to see if there is an `author-link` or not.

lpil commented on _layouts/layout.html:
```diff
@@ -1,4 +1,4 @@
-<!doctype html>
+<!doctype  html>
```
Extra space here now!

lpil reviewed (approved):
Thank you!
